### ISSUE TITLE
'Refresh Standard Runs' must be pressed every time ORCA starts.

### ISSUE THREAD

**Issue by BenLand100** [snoplus/orca#272]
Is there a reason this doesn't happen automatically? I think it should at least attempt to and fail gracefully if there is an issue with the DB because I can't imagine a scenario where the DB was available that you wouldn't click this button. ( @jcaravaca )

**Comment by BenLand100** [comment]
e.g. this fix seems to work fine on the teststand. 
```
diff --git a/Source/Experiments/SNOP/SNOPController.m b/Source/Experiments/SNOP/SNOPController.m
index 18ca838..82ee36c 100755
--- a/Source/Experiments/SNOP/SNOPController.m
+++ b/Source/Experiments/SNOP/SNOPController.m
@@ -240,13 +240,9 @@ snopGreenColor;
     [self refreshRunWordLabels:nil];
     [self runTypeWordChanged:nil];
     //Lock Standard Runs menus at init
-    if([model standardRunType] == nil){
-        [standardRunPopupMenu setEnabled:false];
-        [standardRunVersionPopupMenu setEnabled:false];
-    }
-    else{
-        [self refreshStandardRunsAction:nil];
-    }
+    [standardRunPopupMenu setEnabled:false];
+    [standardRunVersionPopupMenu setEnabled:false];
+    [self refreshStandardRunsAction:nil];
     if(!doggy_icon)
     {
         doggy_icon = [[RunStatusIcon alloc] init];
```

**Comment by jcaravaca** [comment]
That's the way it worked in the past, but it was decided that refreshing the SR should be triggered by the user, so that it doesn't complain when opening the experiment in case the CouchDB settings are not correct. Anyways, I don't have a strong opinion on this so feel free to submit the change if it works at the TS.

**Comment by tlatorre-uchicago** [comment]
However it was set up before it was failing to refresh very often when ORCA started up and logging lots of errors in the log. Has anything changed since then?

**Comment by jcaravaca** [comment]
It should dump an error ('Error querying couchDB, please check the settings are correct and you have connection') if the CouchDB settings are not properly set. I was planning some changes on how errors are handled for the Standard Runs after Eric merges his MTC changes. I can fold this in if we want.

**Comment by tlatorre-uchicago** [comment]
This was fixed in #315.
